src/Synapse/TestHelper/ControllerTestCase.php

```diff
@@ -52,4 +52,24 @@ public function injectMockValidationErrorFormatter(AbstractController $controlle
             ->method('groupViolationsByField')
             ->will($this->returnValue(['foo' => 'bar']));
     }
+
+    public function injectMockTransaction(AbstractController $controller)
```

Comment:
As we might want to use transactions within services as well as controllers, feels like we should have a better base TestCase class with these little general helper methods in it, then have all our test cases extend it instead of the PHPUnit TestCase.

Alternately maybe those general methods should go into a Trait for injection in our Tests instead.  One specific case for this would be WebTestCase, which extends a different PHPUnit class.